### PR TITLE
Update to show the new SSO-based process

### PR DESCRIPTION
**Pull request by bensaunders**
Password synchronisation is no longer needed.
It's no longer necessary to remove a user from all their projects before you delete them.
A different form is used to manage permissions.

### CODE REVIEW

File: articles/openstack/ostack-how-manage-accounts-portal.md [hunk 1]
```diff
@@ -55,7 +53,7 @@ To create a new OpenStack user:
 
     ![New Contact page](images/ptl-new-contact.png)
 
-6. After you've created the user account, you can grant access to OpenStack projects using the steps in [Assigning and revoking OpenStack permissions](#assigning-and-revoking-openstack-permissions).
+6. After you've created the user account, you can grant access to OpenStack projects using the steps in [Assigning and revoking OpenStack permissions](#assigning-and-revoking-openstack-permissions). There may be a delay of a few minutes while the new user's information is sychronised around the system.
```
Comment (suehighmoor) [inline articles/openstack/ostack-how-manage-accounts-portal.md:56]: typo: synchronised

File: articles/openstack/ostack-how-manage-accounts-portal.md [hunk 2]
```diff
@@ -143,16 +108,9 @@ To access OpenStack:
...
 
-If you want to remove an OpenStack user account, you must first revoke the user's permissions to all OpenStack projects before deleting the account.
-
-To remove an OpenStack account:
-
-1. Follow the steps in [Assigning and revoking OpenStack permissions](#assigning-and-revoking-openstack-permissions) to revoke the user's access permissions for all OpenStack projects.
-
-    > [!NOTE]
-    > It takes approximately five minutes for any changes you make within the UKCloud Portal to replicate to OpenStack.
+Removing a user's account in the portal will also remove their access to OpenStack.
```
Comment (suehighmoor) [inline articles/openstack/ostack-how-manage-accounts-portal.md:111]: uppercase P for Portal

File: articles/openstack/ostack-how-manage-accounts-portal.md [hunk 3]
```diff
@@ -143,16 +108,9 @@ To access OpenStack:
...
 
-2. After waiting five minutes for the changes to replicate to OpenStack, return to the *Contacts* list, locate the user again and click the **Remove** button.
+1. Following the instructions above, navigate to the *Contacts* list, locate the user and click the **Remove** button.
```
Comment (suehighmoor) [inline articles/openstack/ostack-how-manage-accounts-portal.md:113]: The section immediately above doesn't show how to get to the Contacts list - it's two sections above, so I'd be tempted to repeat the steps here (plus, if the reader just wants to delete a user and hasn't read the rest of the article, it's annoying to have to scroll up to find some steps, then scroll back down again to carry on with the remaining steps)

File: articles/openstack/ostack-how-manage-accounts-portal.md [hunk 4]
```diff
@@ -22,11 +22,9 @@ toc_mdlink: ostack-how-manage-accounts-portal.md
 
 This guide shows how UKCloud Portal administrators can use the Portal to create user accounts for UKCloud for OpenStack and set access permissions.
```
Comment (suehighmoor) [inline articles/openstack/ostack-how-manage-accounts-portal.md:23]: Could you change guide to article?

File: articles/openstack/ostack-how-manage-accounts-portal.md [hunk 5]
```diff
@@ -73,52 +71,19 @@ To assign or revoke OpenStack permissions:
...
-    > You may need to scroll down the **Permissions** tab to get to the *UKCloud for OpenStack* section.
+    ![Permissions tab for Portal contact](images/ptl-contacts-tab-openstack-permissions.png)
 
-6. For the project to which you want to grant the user access, select the check box for the region in which the project is located. If the project is available in both OpenStack regions (cor00005 or frn00006), you can grant access in just one region or in both.
-
-    > [!NOTE]
-    > If you grant a user access to a region in which the project is not available, the user will not be able to log in to that OpenStack region.
+5. You'll see a list of the OpenStack projects within your organisation. There's a separate entry for each region a project is in.
 
+6. For the project/region to which you want to grant the user access, select the check box.
```
Comment (suehighmoor) [inline articles/openstack/ostack-how-manage-accounts-portal.md:80]: Select the check box for the project and region to which you want to grant the user access.

Comment (bensaunders) [reply articles/openstack/ostack-how-manage-accounts-portal.md]: Done.

File: articles/openstack/ostack-how-manage-accounts-portal.md (hunk 4 above)
Comment (bensaunders) [reply articles/openstack/ostack-how-manage-accounts-portal.md]: Done.

File: articles/openstack/ostack-how-manage-accounts-portal.md (hunk 2 above)
Comment (bensaunders) [reply articles/openstack/ostack-how-manage-accounts-portal.md]: Done.

File: articles/openstack/ostack-how-manage-accounts-portal.md (hunk 1 above)
Comment (bensaunders) [reply articles/openstack/ostack-how-manage-accounts-portal.md]: Done.

File: articles/openstack/ostack-how-manage-accounts-portal.md (hunk 3 above)
Comment (bensaunders) [reply articles/openstack/ostack-how-manage-accounts-portal.md]: Done.

**Comment by bensaunders** [comment]
I've updated the document based on these comments @suehighmoor, thanks very much.

The only one I've left is about typing your password in again to OpenStack - for the moment, I think you still have to do that, because SSO isn't fully used in the Portal yet.

**Comment by suehighmoor** [comment]
Sorry again @bensaunders, have the OpenStack permissions been removed from the general Permissions page now that they have their own tab?

**Comment by bensaunders** [comment]
> have the OpenStack permissions been removed from the general Permissions page now that they have their own tab?

Well, this change to the documentation is all forward-looking. There's a "feature flag" involved, so the behaviour can be switched for everyone at once, or just account by account. A few accounts have been switched over already, and the plan is to switch everyone over next week.

But yes - when this behaviour is switched on, the OpenStack permissions section disappears from the general Permissions page. So I'll make that change too.

**Comment by suehighmoor** [comment]
@jmccukc, I needed to make some more changes because we had a separate article that described how to create a Portal user account for an OpenStack user that included the steps about creating a service request to add the user to the relevant OpenStack projects. As this step is no longer required, I've removed this article as the process for creating a Portal user for OpenStack is now the same as any other Portal user. Can you just confirm that this is good? Thanks!

**Comment by suehighmoor** [comment]
Do not merge until changes are rolled out to all customers (planned for 07/04/2020)

**Comment by jmccukc** [comment]
Updates look good

**Comment by jmccukc** [comment]
further updates look good.

**Pull request merged**